src/components/alert/__snapshots__/__spec__.js.snap

```diff
@@ -0,0 +1,147 @@
+          Object {
+            "minHeight": undefined,
+          }
+        }
+        tabIndex={0}
+      >
+        <div
+          className="carbon-dialog__content"
+        >
+          <div
+            className="carbon-dialog__inner-content"
+            style={Object {}}
+          />
+        </div>
+        <Icon
+          bgSize="small"
+          className="carbon-dialog__close"
+          data-element="close"
+          onBlur={[Function]}
+          tabIndex="0"
+          type="close"
+        />
+      </div>
+    </CSSTransitionGroup>
+    <CSSTransitionGroup
+      component="div"
+      transitionAppear={false}
+      transitionEnter={true}
+      transitionEnterTimeout={500}
+      transitionLeave={true}
+      transitionLeaveTimeout={500}
+      transitionName="modal-background"
+    >
+      <div
+        className="carbon-modal__background"
+      />
+    </CSSTransitionGroup>
+  </div>
+</Portal>
+`;
+
+exports[`Alert tags on internal elements include correct close, subtitle and title tags 1`] = `
+<Portal
+  open={true}
+>
+  <div
+    className="carbon-dialog"
+    data-component="alert"
+  >
+    <CSSTransitionGroup
+      component="div"
+      transitionAppear={false}
+      transitionEnter={true}
+      transitionEnterTimeout={500}
+      transitionLeave={true}
+      transitionLeaveTimeout={500}
+      transitionName="modal"
+    >
+      <div
+        aria-describedby="carbon-dialog-subtitle"
+        aria-labelledby="carbon-dialog-title"
+        className="carbon-dialog__dialog carbon-dialog__dialog--extra-small carbon-alert__alert"
+        data-component="alert"
+        onBlur={[Function]}
+        role="dialog"
+        style={
+          Object {
+            "minHeight": undefined,
+          }
+        }
+        tabIndex={0}
+      >
+        <div
+          className="carbon-dialog__title carbon-dialog__title--has-subheader"
+        >
+          <Heading
+            divider={true}
+            separator={false}
+            subheader="Test"
+            subtitleId="carbon-dialog-subtitle"
+            title="Test"
+            titleId="carbon-dialog-title"
+          />
+        </div>
+        <div
+          className="carbon-dialog__content"
+        >
+          <div
+            className="carbon-dialog__inner-content"
+            style={Object {}}
+          />
+        </div>
+        <Icon
+          bgSize="small"
+          className="carbon-dialog__close"
+          data-element="close"
+          onBlur={[Function]}
+          tabIndex="0"
+          type="close"
+        />
+      </div>
+    </CSSTransitionGroup>
+    <CSSTransitionGroup
+      component="div"
+      transitionAppear={false}
+      transitionEnter={true}
+      transitionEnterTimeout={500}
+      transitionLeave={true}
+      transitionLeaveTimeout={500}
+      transitionName="modal-background"
+    >
+      <div
+        className="carbon-modal__background"
+      />
+    </CSSTransitionGroup>
+  </div>
+</Portal>
```

Comment:
Do we need two snapshots to test this? The snapshot captures the entire component.